doc/new_election/general_election_data_coding_tasks.md

```diff
@@ -6,11 +6,11 @@ In general [db/fixtures/README](../../db/fixtures/README.md) can be a guide. If
 
 ### Switch to general election mode if need be
 
-See [db/seeds.rb](../../db/seeds.rb) and [db/seeds_for_general_election.rb](../../db/seeds_for_general_election.rb) and edit.
+See [db/seeds.rb](../../db/seeds.rb) and optionally edit so that the default election type is general election. This can be overridden using an environment variable (ELECTION_TYPE=general), however, making the edit means other developers don't need to remember to use the envirobment variables.
```

Comment:
```suggestion
See [`db/seeds.rb`](../../db/seeds.rb) and optionally edit so that the default election type is general election. This can be overridden using an environment variable (`ELECTION_TYPE=general`), however, making the edit means other developers don't need to remember to use the environment variables.
```

Reply:
Addressed in #784.